co2eq/purchase/index.js

```diff
@@ -117,7 +118,15 @@ function correctWithParticipants(footprint, participants) {
 }
 
 function extractEur({ costAmount, costCurrency }) {
-  return costAmount && costCurrency ? convertToEuro(costAmount, costCurrency) : null;
+  return costAmount && costCurrency ? convertTo2011Euro(costAmount, costCurrency) : null;
+}
+
+export function convertTo2011Euro(amount, currency) {
```

Comment:
you need to move this function above L120 as else the function is used before it's defined